### DISCUSSION TITLE
Participation Grades

### DISCUSSION THREAD

**Discussion by christian-villandry** [cooperrc/computational-mechanics#203] [Q&A]
Good evening,
Im curious when or if we will know our participation grades, so I can finalize my grade calculations later this week to see if I need to resubmit anything.

Thank you!

**Comment by cooperrc** [comment]
Next week, you'll submit the final participation self assessment. That is your final participation grade. 

Use this as an opportunity to describe what you added to our learning community e.g. discussion posts/answers, working with classmates, meeting with Prof. Cooper, text message groups, ...
